sklearn/metrics/ranking.py

```diff
@@ -173,6 +174,10 @@ def average_precision_score(y_true, y_score, average="macro",
         ``'samples'``:
             Calculate metrics for each instance, and find their average.
 
+    pos_label : int or str (default=1)
+        The label of the positive class. For multilabel-indicator y_true,
```

Comment:
Perhaps clarify that this is only applied to binary classification. We do not, for instance, binarize a multiclass problem using `pos_label`...